native/src/metadata/mod.rs

```diff
@@ -0,0 +1,33 @@
+use std::collections::HashMap;
+use crate::didcomm::Error;
+use crate::proto::okapi::okapi_metadata::{MetadataRequest, MetadataResponse};
+
+impl crate::Metadata {
+    pub fn get_metadata<'a>(request: &MetadataRequest) -> Result<MetadataResponse, Error<'a>> {
+        let env_vars = request.variables.clone();
+
+        // Get the version for sure, and parse it
+        let mut major_ver = 0;
+        let mut minor_ver = 0;
+
+        const PKG_VERSION: Option<&str> = option_env!("CARGO_PKG_VERSION");
+        let pkg_pieces = PKG_VERSION.unwrap().split(".").collect::<Vec<&str>>();
+        if pkg_pieces.len() >= 2 {
+            major_ver = pkg_pieces[0].parse::<i32>().unwrap();
```

Comment:
There's a specific cargo variable to return this, don't parse it from the full version.